deploy/runtime/ui/Financial Solution Mode Single Owner.txt

```diff
@@ -523,28 +523,29 @@ MultilineText
 1
 
 
-2169
-// This form is a copy of Financial Solution Flip Leaseback
+2017
+// This form is a copy of Financial Solution Mode Single Owner
 // so ppa_soln_mode radio button can have different labels
-// to indicate IRR target is for the tax investor for flip 
-// structures. Any updates should be made to both forms.
+// to indicated IRR target is for the tax investor.
+// Any updates should be made to both forms.
 
-on_load{'Financial Solution Mode Single Owner'} = define()
+on_load{'Financial Solution Mode Flip Leaseback'} = define()
```

Comment:
I'm a little confused by the name change here - should this match the top of the file?

Reply:
You are correct this was changed on develop when it should not have been